cmake/Dependencies.cmake

```diff
@@ -92,12 +92,39 @@ endif()
 
 # ---[ Python
 if(BUILD_python)
-  # disable Python 3 search
-  find_package(PythonInterp 2.7)
-  find_package(PythonLibs 2.7)
-  find_package(NumPy 1.7.1)
-  find_package(Boost 1.46 COMPONENTS python)
-
+  if("${python_version}" VERSION_GREATER "2.99")
```

Comment:
I didn't find one. I could just do 3.0.0, since most people will be using 3.2,3.3 or 3.4